### ISSUE TITLE
[feature request] [ux proposal] Min-max linear normalization to be supported in F.normalize

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#107896]
### 🚀 The feature, motivation and pitch

It's useful as one of way of various image / arbitrary normalizations

E.g. supported in OpenCV: https://docs.opencv.org/4.x/d2/de8/group__core__array.html#ggad12cefbcb5291cf958a85b4b67b6149fa9f0c1c342a18114d47b516a88e29822e in the cv2.normalize function: https://docs.opencv.org/4.x/d2/de8/group__core__array.html#ga87eef7ee3970f86906d69a92cbf064bd

Also, related, NumPy supports https://numpy.org/doc/stable/reference/generated/numpy.ptp.html function which computes difference `amax() - amin()`

**Comment by albanD** [comment]
The `ptp` numpy function is not in the Array API standard. @rgommers any insight on this? You think this is a good addition?

**Comment by rgommers** [comment]
> The `ptp` numpy function is not in the Array API standard.

The reasons for that were: 
- It didn't have universal support across array/tensor libraries
- It's easy to implement as `return max(x) - min(x)`, and a JIT compiler may even be able to fuse that one-liner. We tried to avoid such easy-to-implement utility functions in the standard.
- It's not used all that much, doesn't seem like a "core" function

That said, there's nothing wrong with `ptp` (except the name, I'd have liked it to be `peak2peak` or some such thing). So I don't see a problem with adding it in PyTorch if there's enough interest in it. Supporting it as a keyword in `F.normalize` seems nicer than having it as a standalone function to me.
